tests/test_38_organizations.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+include('test_utils.php');
+
+if (!class_exists('bigml')) {
+   include '../bigml/bigml.php';
+}
+
+use BigML\BigML;
+use BigML\BigMLRequest;
+
+class BigMLTestOrganizations extends PHPUnit_Framework_TestCase
+{
+    protected static $api;
+    protected static $api_org;
+    protected static $data_localfile = './data/iris.csv';
+    protected static $project;
+    protected static $test_name;
+
+    public static function setUpBeforeClass() {
+
+       $org = getenv("BIGML_TEST_ORGANIZATION");
+       if ($org == null) {
+          throw new Exception("You need to define env variable " .
+                              "BIGML_TEST_ORGANIZATION to run this test.");
+       }
+
+       self::$api_org =  new BigML(["organization" => $org]);
+       ini_set('memory_limit', '512M');
+       self::$test_name = basename(preg_replace('/\.php$/', '', __FILE__));
+       self::$api_org->delete_all_project_by_name(self::$test_name);
```

Comment:
this is dangerous if anyone runs the tests and depending on the tests name...

Reply:
PHP tests are all like that (and they have been for years and no one complained, maybe because having a project named "test_38_organizations" is unlikely). In any case, I am not against it -- it's more: PHP bindings would beed a thorough refactoring (the code base would shrink by 90%), but I would leave that to the official maintainer.

Reply:
Ok, we can leave that for another PR